doc/release-notes/rl-2505.section.md

```diff
@@ -22,6 +22,8 @@
   The `nixLog` function, which logs unconditionally, was also re-introduced and modified to prefix messages with the function name of the caller.
   For more information, [see this PR](https://github.com/NixOS/nixpkgs/pull/370742).
 
+- `postgresql`'s `pythonSupport` argument has been changed. It is now enabled by default, but to use PL/Python the extension needs to be added explicitly with `postgresql.withPackages`. If you were using `postgresql.override { pythonSupport = true; }` before, change it to `postgresql.withPackages (ps: [ ps.plpython3 ])`. `plperl` and `pltcl` are also available.
```

Comment:
I think the `are also available` isn't explicit enough about the fact that you still have to adjust your Nix code for those cases as well.